src/components/about/PageTemplate.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components/macro'
+
+const PageTemplate = styled.article`
+  @media ${({ theme }) => theme.device.mobile} {
+    width: 80%;
+  }
+
+  max-width: 950px;
+  width: 66%;
+  overflow-y: scroll;
+  height: inherit;
+  margin: 56px auto 0px;
+  overflow-wrap: break-word;
+  h1 {
+    font-family: Lato;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 32px;
```

Comment:
Use [`rems`](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/) for font-size so things are responsive. 

Rems are scalar for the root font size so we can manage responsive sizes for fonts. Let's try to use rems for all font sizes in this PR. The root font style can be seen here: https://github.com/hack4impact-uiuc/falling-fruit/blob/main/src/components/ui/GlobalStyle.js#L59-L65.

Hint: assume 1 rem = 14px, so use `{PIXEL_SIZE / 14}rem`